lib/viewing/HasChildViews.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-var _ = require("lodash");
+var _ = require("underscore");
```

Comment:
yeah. i went back and forth on this. looking at the underscore implementation, it does a simple typeof check too so i figured why bother to change. i'll change it though